src/main/java/frc/team2412/robot/subsystems/LimelightSubsystem.java

```diff
@@ -0,0 +1,213 @@
+package frc.team2412.robot.subsystems;
+
+import edu.wpi.first.math.MathUtil;
+import edu.wpi.first.math.controller.PIDController;
+import edu.wpi.first.math.geometry.Pose2d;
+import edu.wpi.first.math.geometry.Rotation2d;
+import edu.wpi.first.math.util.Units;
+import edu.wpi.first.networktables.NetworkTable;
+import edu.wpi.first.networktables.NetworkTableInstance;
+import edu.wpi.first.wpilibj.shuffleboard.Shuffleboard;
+import edu.wpi.first.wpilibj.shuffleboard.ShuffleboardTab;
+import edu.wpi.first.wpilibj2.command.Command;
+import edu.wpi.first.wpilibj2.command.SubsystemBase;
+import frc.team2412.robot.commands.drivebase.DriveCommand;
+import java.util.function.DoubleSupplier;
+
+public class LimelightSubsystem extends SubsystemBase {
+
+	// CONSTANTS
+
+	private static final PIDController TRANSLATION_PID = new PIDController(10.0, 0, 0);
+	private static final PIDController ROTATION_PID = new PIDController(8.0, 0, 0);
+
+	// meters?
+	public static final double CAMERA_MOUNT_HEIGHT = 0.1143;
+	public static final double CAMERA_ANGLE_OFFSET = 0;
+	public static final double TARGET_HEIGHT = 0.33;
+
+	public static final double GOAL_DISTANCE_FROM_TARGET = 0.7;
+	public static final double GOAL_DISTANCE_FROM_CONE = 0.3;
+	public static final double GOAL_DISTANCE_FROM_CUBE = 0.3;
+
+	// MEMBERS
+
+	NetworkTable networkTable;
+
+	String currentPoseString;
+	String targetPoseString;
+
+	// network tables
+
+	// CONSTRUCTOR !
+	public LimelightSubsystem() {
+
+		// broadcast 10.24.12.227
+
```

Comment:
? Not sure what this is for. The Limelight should have a static IP address, likely 10.24.12.11 (Orange Pi is at 10.24.12.12)

Reply:
Changed the comment to describe the location of the limelight config and added proper address